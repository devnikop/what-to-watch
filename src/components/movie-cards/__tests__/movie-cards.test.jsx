import React from "react";
import renderer from "react-test-renderer";

import MovieCards from "../movie-cards.jsx";

const mock = {
  movies: [
    {
      id: 1,
      src: `source_11`,
      title: `Fantastic Beasts`,
    },
    {
      id: 2,
      src: `source_22`,
      title: `Aviator`,
    },
    {
      id: 3,
      src: `source_33`,
      title: `Bohemian rhapsody`,
    },
  ],
};

it(`snapshot`, () => {
  const { movies } = mock;

  const tree = renderer.create(<MovieCards movies={movies} />).toJSON();
  expect(tree).toMatchSnapshot();
});
