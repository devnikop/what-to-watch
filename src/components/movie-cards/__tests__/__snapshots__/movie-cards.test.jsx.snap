// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Fantastic Beasts"
        height="175"
        src="source_11"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Fantastic Beasts
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Aviator"
        height="175"
        src="source_22"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Aviator
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Bohemian rhapsody"
        height="175"
        src="source_33"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Bohemian rhapsody
      </a>
    </h3>
  </article>
</div>
`;
